src/types/transaction_request.rs

```diff
@@ -128,4 +150,30 @@ mod tests {
 }"#
         );
     }
+
+    #[test]
+    fn should_deserialize_transaction_request() {
+        let tx_request = TransactionRequest {
```

Comment:
There is already a test for serialization, I think for simplicity of the test I'd prefer to just use a raw JSON as `serialized`.